### PR TITLE
Issue with learning environment minikube

### PR DESCRIPTION
**Pull request by RA489**
Issue with learning environment minikube
fixes https://github.com/kubernetes/website/issues/15728

### CODE REVIEW

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-15845--kubernetes-io-master-staging.netlify.com/docs/setup/learning-environment/minikube/

File: content/en/docs/setup/learning-environment/minikube.md [hunk 1]
```diff
@@ -55,6 +55,13 @@ This brief demo guides you on how to start, use, and delete Minikube locally. Fo
     ```shell
     kubectl run hello-minikube --image=k8s.gcr.io/echoserver:1.10 --port=8080
     ```
+    {{< note >}}
+    kubectl run --generator=deployment/apps.v1 is DEPRECATED and will be removed in a future version. Use kubectl run --generator=run-pod/v1 or kubectl create instead. e.g.
+
+    ```shell
+    kubectl create deployment hello-minikube --image=k8s.gcr.io/echoserver:1.4
+    ```
+    {{< note >}}
```
Comment (kbhawkey) [inline content/en/docs/setup/learning-environment/minikube.md]: hello @RA489, would you fix the closing note shortcode. Thanks!
{{< /note >}}

Comment (RA489) [reply content/en/docs/setup/learning-environment/minikube.md]: sure

Comment (RA489) [reply content/en/docs/setup/learning-environment/minikube.md]: @kbhawkey fixed!

**Comment by zacharysarah** [comment]
Resolved by #16533. 

/close

**Pull request closed without merging**